app/models/user_track.rb

```diff
@@ -29,7 +29,7 @@ def self.for!(user_param, track_param)
     )
   end
 
-  def self.for(user_param, track_param, external_if_missing: false)
+  def self.for(user_param, track_param, external_if_missing: true)
```

Comment:
I suggest removing `external_if_missing` entirely now.

Reply:
Agreed. I wanted to save that for last :0